smbios/system_information.go

```diff
@@ -13,6 +13,32 @@ type SystemInformationStructure struct {
 	smbios.Structure
 }
 
+type wakeUpType int
+
+const (
+	typeReserved wakeUpType = iota
+	typeOther
+	typeUnknown
+	typeAPMTimer
+	typeModemRing
+	typeLANRemote
+	typePowerSwitch
+	typePCIPME
+	typeACPowerRestored
+)
```

Comment:
Looks like this isn't used, but I think we could, and it would be really nice. Take a look at https://github.com/talos-systems/talos/blob/47608fb874ba63f002767e5422654858b19f8370/internal/app/machined/pkg/runtime/mode.go#L11-L32 for a guideline on how to add a string method to `wakeUpType`. Then we can do something like 

```
t := wakeUpType(s.Formatted(20))

return t.String()
```

Reply:
Yep, good call. I've updated in amended commit.